sensei-lms.php

```diff
@@ -3,7 +3,7 @@
  * Plugin Name: Sensei LMS
  * Plugin URI: https://senseilms.com/
  * Description: Share your knowledge, grow your network, and strengthen your brand by launching an online course.
- * Version: 4.20.0
+ * Version: 4.20.1
  * Author: Automattic
  * Author URI: https://automattic.com
  * License: GPL version 2 or later - http://www.gnu.org/licenses/old-licenses/gpl-2.0.html
```

Comment:
In this file you can find `Copyright 2013-2022 Automattic`. Should we update it to `2013-2024`?

Reply:
Is it okay if we just update it or there's a process/prerequisite to update it?

Reply:
I don't know of a process around it, last time I just updated it.

Reply:
Hi @m1r0 👋 I just discussed a bit with Donna and decided to remove the text entirely. Looks like woo doesn't have it. Can you kindly give it another look? Thanks!

https://github.com/Automattic/sensei/pull/7433/commits/406e9edc2de67702df41bc9bd4f8a993e21cd607

Reply:
Actually, I'll go ahead and merge it :p Please leave a comment if you see something in need of addressing